docs/terms/oidc.md

```diff
@@ -0,0 +1,8 @@
+---
+id: oidc
+title: OpenID Connect (OIDC)
+hoverText: An identity layer built on top of the OAuth 2.0 framework. [OIDC spec](https://openid.net/specs/openid-connect-session-1_0.html)
```

Comment:
Did you intend to link to OIDC Session Management or OIDC Core? https://openid.net/specs/openid-connect-core-1_0.html﻿

Reply:
good catch!